google-api-go-generator/gen.go

```diff
@@ -2061,8 +2061,11 @@ func (meth *Method) generateCode() {
 		}
 		pn(`})`)
 	}
-
-	pn("return gensupport.SendRequest(c.ctx_, c.s.client, req)")
+	if meth.supportsMediaUpload() {
```

Comment:
This will add retries for 60 different api calls, maybe we should just scope this to storage(which has this on one api) for now until retires are evaluated more across the board. Thoughts?